source/index.html.md

```diff
@@ -439,6 +439,26 @@ You can use the `every` to check if all the elements satisfies specific conditio
 
 You can use the `in expression` to check if a given value is matched by a specified range.
 
+## boolean expression(and, or)
+
+> Examples
+
+```FEEL,commented
+true and true   //➔ true
+true and false and null   //➔ false
+true and null and true   //➔ null
+true or false or null   //➔ true
+false or false   //➔ false
+false or null or false  //➔ null
+true or false and false   //➔ true
+(true or false) and false   //➔ false
+```
```

Comment:
I can't get github to properly handle a github suggestion, one reason more after ` ``` ` do maintain an empty newline, to avoid syntax highlight confusion